JobConfig/cosmic/cosmic_s2_resampler.fcl

```diff
@@ -0,0 +1,48 @@
+// Re-sample cosmics from stage-1
+// Yuri Oksuzian, 2020
+
+#include "JobConfig/primary/primary.fcl"
+#include "JobConfig/cosmic/epilog.fcl"
+
+// Bring in info from the previous stage
+physics.filters.dsResample: {
```

Comment:
Is there a reason dsResample isn't defined in prolog?

Reply:
Are you suggesting to put dsResample in cosmic/prolog.fcl
I'm not sure what's the purpose of doing that.
dsResample is only used for stage-2.